Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml

```diff
@@ -553,3 +553,14 @@
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpskirt/janimaid.rsi
     
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpskirtBlueShield
+  name: blue shield
```

Comment:
You have to actually write "Blueshield Jumpskirt" or something. Blue Shield as the item name is unacceptable.

Reply:
ok